app/src/main/java/pl/futurecollars/invoicing/App.java

```diff
@@ -1,12 +1,36 @@
 package pl.futurecollars.invoicing;
 
-public class App {
+import java.math.BigDecimal;
+import java.time.LocalDate;
+import java.util.List;
+import pl.futurecollars.invoicing.db.Database;
+import pl.futurecollars.invoicing.db.memory.InMemoryDatabase;
+import pl.futurecollars.invoicing.model.Company;
+import pl.futurecollars.invoicing.model.Invoice;
+import pl.futurecollars.invoicing.model.InvoiceEntry;
+import pl.futurecollars.invoicing.model.Vat;
+import pl.futurecollars.invoicing.service.InvoiceService;
 
-    public String getGreeting() {
-        return "Hello World!";
-    }
+public class App {
 
     public static void main(String[] args) {
-        System.out.println(new App().getGreeting());
+        Database db = new InMemoryDatabase();
```

Comment:
wszystko tutaj jest ok, ale staraj się nie kopiować gotowego rozwiązania, tylko pisać samemu - dużo więcej się nauczysz (nawet napisanie tego samego we własny sposób da więcej)